routes/users.js

```diff
@@ -1,9 +1,83 @@
 var express = require('express');
 var router = express.Router();
+var passport = require('passport');
+var LocalStrategy = require('passport-local').Strategy;
 
-/* GET users listing. */
-router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+//custom modules
+var User = require('../models/users');
+var Passport = require('./passport');
+
+
+//signup
+router.get('/register', function (req, res) {
+  res.render('register');
 });
 
+//login
+router.get('/login', function (req, res) {
+  res.render('login');
+});
+
+router.post('/register',function (req, res) {
+  //console.log(req.body)
```

Comment:
Remove console.logs if it doesnt describe anything.

Reply:
Done